internal/input/ledgers.go

```diff
@@ -68,7 +68,12 @@ func GetLedgers(start, end uint32, limit int64, env utils.EnvironmentDetails, us
 			},
 		}
 
-		ledgerSlice = append(ledgerSlice, ledger)
+		ledgerLCM := utils.HistoryArchiveLedgerAndLCM{
```

Comment:
Because of the new LCM fee field we need to pass in the `historyarchive.Ledger` as well as the `LedgerCloseMeta`. This should be eventually refactored in the future to where `input/ledger` and `input/assets` don't use `historyarchive.Ledger` and just use LCM